src/global.json

```diff
@@ -1,4 +1,9 @@
 {
+	"sdk":
+	{
+		"version": "6.0.100",
+		"rollForward": "latestFeature"
+	},
```

Comment:
This is to make the repo easier to work with locally. It won't use the .NET 8 SDK which throw for net6.0-ios and android targets. It reflects the 6.0.x semantics in the GitHub action.

Reply:
I've been adding something similar to my local work directory.

I'm a little bit tempted to deprecate net6.0 next week.

But net6.0 is still in support for another year.

But the net6.0 mobile targets are a special case.

Reply:
While I'm confessing my temptations, I'd love to deprecate support for .NET Framework too, but that's a whole different can of worms.  :-)

Reply:
In our fork, I compromised by supporting the base net6.0 TFM, but requiring net7.0-android and -ios.

Reply:
I still want .NET Framework support, but only for VS.

Reply:
Speaking broadly, not just about SQLite...

I always wonder how the support story for .NET Framework would be different without VS in the picture.  I assume it'll probably never move to [.NET Core and its descendants].  Lots of enterprise apps are in the same boat, but VS is a special case.

No comments expected -- I'm just musing a bit.